src/main/webapp/config.js

```diff
@@ -0,0 +1,3 @@
+var config = {
+  API_KEY : "AIzaSyA_KkkKwMJRz4Zg4zn_bcIf20BTAbmZ6yI"
```

Comment:
As discussed, please remove keys from git and get a new one.

Reply:
Solution TBD, will fix in future branch, if necessary.

Reply:
I'm approving since Erick said we can keep this secure by the whitelisting properties of the API Key.